lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -8,6 +8,16 @@ def define
 
       config = Calculators::BenefitCapCalculatorConfiguration.new
 
+      # Routing question
+      multiple_choice :choose_cap_to_calculate? do
+        option :default
+        option :future
+
+        next_node do
+          question :receive_housing_benefit?
+        end
+      end
+
       # Q1
       multiple_choice :receive_housing_benefit? do
         option :yes
```

Comment:
With hindsight, it probably would've lead to slightly easier to understand commits if I'd split this into two. Having said that, I'm confident that I understand the changes in this large commit so am not too worried about it being combined.

Irrespective, I should've removed the TODO from the commit note. Sorry!